front/app/containers/Admin/projects/project/index.tsx

```diff
@@ -64,7 +65,7 @@ const AdminProjectsProjectIndex = ({
   const tabs: IPhaseTab[] = selectedPhase
     ? getTabs(selectedPhase, featureFlags, formatMessage).map((tab) => ({
         ...tab,
-        url: `${baseTabsUrl}/phases/${selectedPhase.id}/${tab.url}`,
+        url: `${baseTabsUrl}/phases/${selectedPhase.id}/${tab.url}` as RouteType,
       }))
```

Comment:
@EdwinKato Same thought here - using the casting means we could still enter invalid URLs right? :thinking: 

E.g. If I add more to the end of the URL:
![image](https://github.com/CitizenLabDotCo/citizenlab/assets/33987955/28a0557b-3e36-4ab1-a6d3-ac50815d4a2c)